src/tracing/tracing.service.ts

```
@@ -25,7 +25,27 @@ export class TracingService implements OnModuleInit, OnApplicationShutdown {
       'JAEGER_HOST',
       'localhost',
     );
-    const jaegerPort = this.configService.get<number>('JAEGER_PORT', 6831);
+
+    // Ensure port is properly parsed as a number
+    let jaegerPort: number = 6831; // Default port
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)